sdk/wallet-standard/src/wallet.ts

```diff
@@ -1,6 +1,19 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { fromB64, toB64 } from '@mysten/sui.js/utils';
+import type { Wallet, WalletWithFeatures } from '@wallet-standard/core';
+import { getWallets } from '@wallet-standard/core';
+
+import { isWalletWithRequiredFeatureSet } from './detect.js';
+
+import './features/index.js';
```

Comment:
Does this have side-effects or something?

Reply:
No, not sure why that was there